meteor/sofie-server/data-access/repositories/mongo/mongo-entity-converter.ts

```diff
@@ -114,6 +114,17 @@ export class MongoEntityConverter {
 		return mongoRundowns.map(this.convertRundown)
 	}
 
+	public convertToMongoRundown(rundown: Rundown): Partial<MongoRundown> {
```

Comment:
Partial was also more-so a placeholder, so i didn't have to assign each and every attribute. From my understanding that is going to be done during the task to implement the `saveRundown(...)` method, which currently just throws an error saying 'Not implemented'

Reply:
The "add as we go" approach is in regards to which variables we need on our entities. E.g. we add `outTransition` to `Part` when we get to implement logic that requires `outTransition`.

For the converter you need to fill out all the variables that we currently have on our entities.

By using `Partial` you now have to make all your variables be `const mongoRundown: Partial<MongoRundown>`. So whoever then comes and finishes the `MongoEntityConverter` and changes the type back to `MongoRundown` (as it should be) now has to go find all your usages to refactor the type as well.

Reply:
As of writting this comment, there is only one place where it would need to be changed, had i added the type explicitly.
As the type is being infered at the moment, there wouldn't be anywhere that'd need to be changed, should the return types of the convert method change.

With that said, i get what is being said, and will be changing the types to not be partial.